Webpack plugin fails on compile error in Docker container

<!--
Thanks for filing an issue!

If you have a question, consider searching and asking in other Electron community sites,
so that more people have a chance to help you: https://github.com/electron/electron#community

Specific issues about the templates that are available by default when creating a new
Electron Forge project should be filed in its GitHub repository:
https://github.com/electron-userland/electron-forge-templates

Please check off all of the steps as they are completed by replacing [ ] with [x].
-->

* [x] I have read the [contribution documentation](https://github.com/electron-userland/electron-forge/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

**Please describe your issue:**
I'm trying to get electron-forge 6 working in a Docker environment. When I attempt to run the following minimal example, `electron-forge start` fails with error 
```
Cannot read property 'toString' of null
TypeError: Cannot read property 'toString' of null
    at Watching.toString [as handler] (/usr/src/webpack-demo/node_modules/@electron-forge/plugin-webpack/src/WebpackPlugin.ts:363:27)
```


`Dockerfile`:
```
FROM node:8.16.0-stretch

RUN npm install -g npx

WORKDIR /usr/src
RUN npx create-electron-app webpack-demo --template=webpack
WORKDIR /usr/src/webpack-demo

RUN apt-get update
RUN apt-get install libnss3 libgtk-3-0 libxss1 libasound2 -y

ENV DEBUG electron-forge:*

USER node

CMD ["npm", "start"]
```


<!-- For bugs and questions, please answer these questions to the best of your ability. -->

**Console output when you run `electron-forge` with the environment variable `DEBUG=electron-forge:*`. (Instructions on how to do so [here](https://www.npmjs.com/package/debug#usage)). Please include the stack trace if one exists.**

Output from running `docker build -t webpack-template . && docker run webpack-template
` with the above `Dockerfile`

```
> webpack-demo@1.0.0 start /usr/src/webpack-demo
> electron-forge start

WARNING: DEBUG environment variable detected.  Progress indicators will be sent over electron-forge:lifecycle
2019-07-12T18:30:55.453Z electron-forge:async-ora Process Started: Checking your system
2019-07-12T18:30:55.461Z electron-forge:check-system checking system, create ~/.skip-forge-system-check to stop doing this
2019-07-12T18:30:56.445Z electron-forge:async-ora Process Succeeded: Checking your system -- after 993ms
WARNING: DEBUG environment variable detected.  Progress indicators will be sent over electron-forge:lifecycle
2019-07-12T18:30:57.412Z electron-forge:async-ora Process Started: Locating Application
2019-07-12T18:30:57.415Z electron-forge:project-resolver searching for project in: /usr/src/webpack-demo
2019-07-12T18:30:57.422Z electron-forge:project-resolver electron-forge compatible package.json found in /usr/src/webpack-demo/package.json
2019-07-12T18:30:57.424Z electron-forge:async-ora Process Succeeded: Locating Application -- after 12ms
2019-07-12T18:30:57.437Z electron-forge:require-search searching [ '@electron-forge/plugin-webpack',
  '/usr/src/webpack-demo/@electron-forge/plugin-webpack',
  '/usr/src/webpack-demo/node_modules/@electron-forge/plugin-webpack' ] relative to /usr/src/webpack-demo
2019-07-12T18:30:57.441Z electron-forge:require-search testing @electron-forge/plugin-webpack
2019-07-12T18:30:58.190Z electron-forge:plugin:webpack hooking process events
2019-07-12T18:30:58.243Z electron-forge:async-ora Process Started: Preparing native dependencies
2019-07-12T18:30:58.814Z electron-forge:async-ora Process Succeeded: Preparing native dependencies -- after 571ms
2019-07-12T18:30:58.819Z electron-forge:plugins plugin: "webpack" has taken control of the start command
2019-07-12T18:30:58.838Z electron-forge:async-ora Process Started: Compiling Main Process Code

An unhandled exception has occurred inside Forge:
Cannot read property 'toString' of null
TypeError: Cannot read property 'toString' of null
    at Watching.toString [as handler] (/usr/src/webpack-demo/node_modules/@electron-forge/plugin-webpack/src/WebpackPlugin.ts:363:27)
    at Watching._done (/usr/src/webpack-demo/node_modules/webpack/lib/Watching.js:95:9)
    at compiler.emitAssets.err (/usr/src/webpack-demo/node_modules/webpack/lib/Watching.js:52:27)
    at emitFiles (/usr/src/webpack-demo/node_modules/webpack/lib/Compiler.js:319:20)
    at /usr/src/webpack-demo/node_modules/mkdirp/index.js:35:29
    at /usr/src/webpack-demo/node_modules/mkdirp/index.js:47:53
    at FSReqWrap.oncomplete (fs.js:152:21)
2019-07-12T18:30:59.732Z electron-forge:plugin:webpack handling process exit with: { cleanup: true }
2019-07-12T18:30:59.733Z electron-forge:plugin:webpack cleaning webpack watcher
2019-07-12T18:30:59.734Z electron-forge:plugin:webpack stopping logger
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webpack-demo@1.0.0 start: `electron-forge start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webpack-demo@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2019-07-12T18_30_59_803Z-debug.log
```


**What command line arguments are you passing?**
None

**What does your `config.forge` data in `package.json` look like?**

I'm using the webpack template from `npx create-electron-app webpack-demo --template=webpack` as is with no modifications.

**Please provide either a failing minimal testcase (with a link to the code) or detailed steps to
reproduce your problem. Using `electron-forge init` is a good starting point, if that is not the
source of your problem.**

1. Have Docker installed
1. Copy minimal `Dockerfile` above to a new dir
1. In the dir, run `docker build -t webpack-template . && docker run webpack-template`

--- a/packages/plugin/webpack/src/WebpackPlugin.ts
+++ b/packages/plugin/webpack/src/WebpackPlugin.ts
@@ -360,7 +360,7 @@ Your packaged app may be larger than expected if you dont ignore everything othe
         const compiler = webpack(await this.getMainConfig());
         const [onceResolve, onceReject] = once(resolve, reject);
         const cb: webpack.ICompiler.Handler = (err, stats) => {
-          if (tab) {
+          if (tab && stats) {
             tab.log(stats.toString({
               colors: true,
             }));
@@ -443,9 +443,11 @@ Your packaged app may be larger than expected if you dont ignore everything othe
             const tab = logger.createTab(`${entryPoint.name} - Preload`);
             const [onceResolve, onceReject] = once(resolve, reject);
             const cb: webpack.ICompiler.Handler = (err, stats) => {
-              tab.log(stats.toString({
-                colors: true,
-              }));
+              if (stats) {
+                tab.log(stats.toString({
+                  colors: true,
+                }));
+              }
 
               if (err) return onceReject(err);
               return onceResolve();

